portfolio.html

```diff
@@ -68,7 +70,9 @@ <h3>Project 4</h3>
 		
 		<section class="sm-section">
 			<a id="contact-me"></a>
-			<img class="contact-image" src="file:///Users/Max/Downloads/1485969_796556610358000_706157516_o%20(2).jpg" alt="contact me image">
+			<div>
+				<img class="contact-image" src="file:///Users/Max/Downloads/1485969_796556610358000_706157516_o%20(2).jpg" alt="contact me image">
```

Comment:
Might be a good idea to move this image into the product folder as it won't be avaliable for anyone else.